run_cumulus_task.py

```diff
@@ -59,14 +72,7 @@ def run_cumulus_task(
         try:
             return task_function(cumulus_message, context, **taskargs)
         except Exception as exception:
-            name = exception.args[0]
-            if isinstance(name, str) and 'WorkflowError' in name:
-                cumulus_message['payload'] = None
-                cumulus_message['exception'] = name
-                logger.error('WorkflowError')
-                return cumulus_message
-            logger.error(exception)
-            raise
+            return handle_task_exception(exception, cumulus_message, logger)
```

Comment:
This is just a little cleanup to reuse the same exception handling logic rather than repeating ourselves